README.md

```diff
@@ -1,6 +1,6 @@
 This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
 
-## Getting Started
+## Get ting Started
```

Comment:
why did you change this? :)

Reply:
you can't commit a brand new branch without changes, that"s for the intial commit of the branch.